_config.yml

```diff
@@ -8,8 +8,8 @@ permalink:        pretty
 # Setup
 title:            MDAnalysis
 name:             MDAnalysis
-tagline:          A Python package for the analysis of molecular dynamics simulations.
-description:      A Python package for the analysis of molecular dynamics simulations.
+tagline:          A Python package for the analysis of molecular simulations.
```

Comment:
This is <s>probably</s> definitely my 2 am brain being silly here, but would "molecular simulation data" be better?

Also analysis seems to be short selling MDA (I think nearly 50% of my use nowadays is pre-simulation processing), would "handling and analysis" or something like that be better?

Reply:
If you can roll it all (pre and post) into a handy slogan – please do!

Reply:
How about?

```suggestion
tagline:          A Python package for the manipulation and analysis of molecular simulations data.
```